CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 5.15.3 / 4.15.3 [NOT PUBLISHED YET]
+
+-   Fixes #2250 - extendObservable: allows it to be used on an existing property [#2250](https://github.com/mobxjs/mobx/pull/2250)
```

Comment:
Move this into a previous unpublished version please.

Reply:
Oh, put it in 5.15.2 you mean?